src/chart/helper/Symbol.ts

```diff
@@ -80,7 +81,7 @@ class Symbol extends graphic.Group {
         );
 
         symbolPath.attr({
-            z2: 100,
```

Comment:
1. `Symbol` is used not only by markPoint, but also by line series, scatter series, etc.
In the current PR, only markPoint allows users to set z2, which results in `z2: 100` being missing in other cases. Is it intentional? If not, backward compatibility should be maintained.

2. If users do not specify `z2`, here a undefined-z2 will be set to an zr element, which is error-prone (z2 is not meant to be a non-number yet in zrender) and might degrade the performance (theoretically).

Summary: I think every place a `z2` is about to set to a zr element, use a default number value to make sure it's a number.
e.g.,
```js
symbolPath.attr({
    zr: retrieve2(z2, 100),
    ...
})
```
```js
graphic.updateProps(polygon, {
    retrieve2(z2, 0),
    // ...
})
```